### ISSUE TITLE
Window size issue

### ISSUE THREAD

**Issue by Bonicx** [ocornut/imgui#1348]
Removed everything due to lack of activity/response.

**Comment by ocornut** [comment]
I don't understand your question, please clarify.
Your application knows the screen resolution and therefore you can use SetNextWindowSize(screen_resolution) if you need.

**Comment by Bonicx** [comment]
EditorUpdate loop function
=================================================================
// Setup display frameSize (every frame to accommodate for window resizing)
	RECT rect2;
	GetClientRect(*g_Window, &rect2);

	io.DisplaySize = ImVec2((float)rect2.right - rect2.left, (float)rect2.bottom - rect2.top);
=================================================================

This is the window part
=================================================================
ImVec2 size = io.DisplaySize;

	//ImGui::SetWindowSize(ImVec2(size.x/4, size.y/4));
	ImGui::SetNextWindowSize(ImVec2(size.x/5, size.y/5));

	if (!Open_Close_InspectorWindow)return;
	ImGuiWindowFlags window_flags = 0;
	window_flags = ImGuiWindowFlags_AlwaysVerticalScrollbar | ImGuiWindowFlags_NoMove | ImGuiWindowFlags_NoCollapse;

	//Generates the ImGui window for Inspector
	ImGui::Begin("Inspector", 0, window_flags);
=================================================================

I'm trying different ways to compute the window to scale automatically but it won't fit inside the window on my home screen 
![image](https://user-images.githubusercontent.com/32429706/31053202-d3b446e0-a6ca-11e7-8680-e01637d4842e.png)

I have been developing the editor imgui with my school computer but I have to figure some way to let it scale according to different resolution.

**Comment by Bonicx** [comment]
Updated with images, to show what I mean look at the bottom edges, the 1st picture does not have the resize draggable arrow because it is not scaled to the monitor i'm using at home.

While the second image is the one I desire to have when using any resolution. I do not want to hard code it that is why I would like to know how do I compute it?

EditorUpdate calls my Inspector window draw function so if i tried to compute like what you have seen in the code with divison of a certain number I encountered problem such as, unable to run-time resize due to the update function is updating the window per frame.
